core/src/main/resources/hudson/model/Run/console.jelly

```diff
@@ -50,38 +50,7 @@ THE SOFTWARE.
         ${%Console Output}
       </t:buildCaption>
 
-      <j:set var="threshold" value="${h.getSystemProperty('hudson.consoleTailKB')?:'150'}" />
-      <!-- Show at most last 150KB (can override with system property) unless consoleFull is set -->
-      <j:set var="offset" value="${empty(consoleFull) ? it.logText.length()-threshold*1024 : 0}" />
-      <j:choose>
-        <j:when test="${offset > 0}">
-          ${%skipSome(offset/1024,"consoleFull")}
```

Comment:
This seems to have orphaned the message https://github.com/jenkinsci/jenkins/blob/72f78abae533b0bfe9782df8618123b18a53a37c/core/src/main/resources/hudson/model/Run/console.properties#L23